src/modules/Api.js

```diff
@@ -0,0 +1,39 @@
+const url = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/0pNrfjBwCKXqbjF87mV/scores/';
+const addData = function (name, score) {
+  const DataToSend = {
+    user: name,
+    score,
+  };
+  fetch(url, {
```

Comment:
[REQUIRED] It is a requirement to use async and await in data fetching in this milestone. Kindly fix this.